backend/app/api_cheese.py

```
@@ -619,10 +619,10 @@ def _background_push_worker(app, user_id, tracker_id, added_slots, removed_slots
             # Send notifications AFTER successful commit
             if notifications_outbox:
                 try:
-                    from firebase_admin import messaging
-                    for tokens, messages in notifications_outbox:
-                        messaging.send_each(messages)
-                        logging.info(f"[CHEESE_DEBUG] Sent collision push notification after commit to user {user_id}")
+                    for messages in notifications_outbox:
+                      for (subscription_info, data) in messages:
+                        webpush(subscription_info, data, VAPID_PRIVATE_KEY_FILE, get_vapid_claims())
+                    logging.info(f"[CHEESE_DEBUG] Sent collision push notification after commit to user {user_id}")
                 except Exception as p_err:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)